### ISSUE TITLE
Changing name of relation causes infinite wait

### ISSUE THREAD

**Issue by heymartinadams** [prisma/prisma#2536]
# Bug Report

**Current behavior**
Using `prisma v1.8.3`. After changing the name of a relation inside `datamodel.graphql`, `prisma deploy` signs off on changes, but then keeps `Applying changes (12/9)` until such time that the token expires.
Here’s the complete error message:
```
ERROR: Authentication token is invalid: Token can't be decoded: The token is expired since 2018-05-29T15:39:59Z

{
  "data": null,
  "errors": [
    {
      "locations": [
        {
          "line": 2,
          "column": 11
        }
      ],
      "path": [
        "migrationStatus"
      ],
      "code": 3015,
      "message": "Authentication token is invalid: Token can't be decoded: The token is expired since 2018-05-29T15:39:59Z",
      "requestId": "__PATH__:management:__PROJECT_ID__"
    }
  ],
  "status": 200
}
```

**Reproduction**
For example, take the following datamodel:
```
type AffiliateCampaign @model {
  requirements: AffiliateRequirement @relation(name: "RequirementsForEachCampaign")
}

type AffiliateRequirement @model {
  campaign: AffiliateCampaign @relation(name: "RequirementsForEachCampaign")
}
```
If you change the name `RequirementsForEachCampaign`, this happens.

**Comment by heymartinadams** [comment]
Wonder if it’s related to this [closed issue](https://github.com/prismagraphql/prisma/issues/2030).

**Comment by heymartinadams** [comment]
Definitely needs to be kept open, you silly bot. Still throwing same error! 😀

**Comment by VladimirTambovtsev** [comment]
Have the same issue. Any updates?

**Comment by mavilein** [comment]
Thanks for reporting the issue and your thorough description! 🙏 This has been fixed in `1.18.1`.

**Comment by heymartinadams** [comment]
See https://github.com/prisma/prisma/issues/2906#issuecomment-431235176
